schemas/quality_performance_score.yaml

```diff
@@ -0,0 +1,50 @@
+---
+quality_performance_score:
+  anyOf:
+  - type: object
```

Comment:
(question) So the response could be like this:
```
{
  "error": {
    "message": "error message",
    "code": "ERROR_CODE"
  }
}
```

or like this?
```
{
  "data": [
    "translations": [
      {
        "engine": "engine_type",
        "score": 0.12,
        "id": "segment_id"
      }
    ]
  ],
  "errors": [
    {
      "message": "error message",
      "code": "ERROR_CODE",
      "id": "some_id"
    }
  ]
```

How come the error objects don't match? The former doesn't have id, while the latter does. Or does the id refer to the translation id?

Reply:
Ah, the id does refer to the translation id passed in. I saw from https://github.com/phrase/phrase/pull/12451